cordova-plugin-outline/android/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -173,6 +173,16 @@ public static TunnelConfig makeTunnelConfig(final String tunnelId, final JSONObj
     } catch (JSONException e) {
       LOG.fine("Tunnel config missing name");
     }
+    try {
+      String prefix = config.getString("prefix");
```

Comment:
Don't you need to check if prefix is present?

Reply:
This try-catch provides that check, as with `name` above.